recipes/syft/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set goname = "syft" %}
+{% set version = "0.89.0" %}
+
+
+package:
+  name: {{ goname|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  - folder: src
+    url: https://github.com/anchore/syft/archive/v{{ version }}.tar.gz
+    sha256: e8b4bf1481f3101adb2881f35f95528cceb58470f1c568c7c05c2864031dfdaa
+
+build:
+  number: 0
+  script:
+    - pushd src
+    - go build -ldflags -v -o $PREFIX/bin/syft  # [not win]
+    - go build -ldflags -v -o %LIBRARY_BIN%\syft.exe  # [win]
+    - bash -c "mkdir -p ${PREFIX}/share/syft/shell; cp -v shell/* ${PREFIX}/share/syft/shell/"
+    - bash -c "mkdir -p ${PREFIX}/share/syft/plugin; cp -v plugin/* ${PREFIX}/share/syft/plugin/"
+    - go-licenses save . --save_path ../library_licenses
+    # Clear out cache to avoid file not removable warnings
+    - chmod -R u+w $(go env GOPATH) && rm -r $(go env GOPATH)  # [unix]
+
+requirements:
+  build:
+    - {{ compiler('go-nocgo') }}
+    - go-licenses
+
+test:
+  commands:
+    - syft --version
+
+about:
+  home: https://github.com/anchore/syft
+  license: Apache-2.0
+  license_family: Apache
+  license_file:
+    - src/LICENSE
+    - library_licenses/
```

The go-licenses command fails because of an unknown license in one of its dependencies. Let's deal with that later.

```suggestion
    # Clear out cache to avoid file not removable warnings
    - chmod -R u+w $(go env GOPATH) && rm -r $(go env GOPATH)  # [unix]

requirements:
  build:
    - {{ compiler('go-nocgo') }}
    - go-licenses

test:
  commands:
    - syft --version

about:
  home: https://github.com/anchore/syft
  license: Apache-2.0
  license_family: Apache
  license_file:
    - src/LICENSE
```

---

We won't merge though without the licenses.

---

> We won't merge though without the licenses.

100% agreed. I just want to check if the build succeeds.